packages/slate-editor/src/modules/popper-options-context/types.ts

```diff
@@ -5,7 +5,7 @@ export interface Context {
     modifiers?: {
         preventOverflow?: PreventOverflowModifier['options'];
     };
-    placement?: 'left-start' | 'right-start';
+    placement?: 'left' | 'right';
```

Comment:
That was the original issue, allowing only `*-start` makes popper element to jump from top to bottom. Example: https://codesandbox.io/s/thirsty-taussig-8r8rzo?file=/src/index.js